iacrcc/iacrcc.cls

```diff
@@ -1,5 +1,5 @@
-\def\fileversion{0.34}
-\def\filedate{2023/07/28}
+\def\fileversion{0.35}
```

Comment:
we typically bump the version in iacrdoc.tex at the same time. Just make that change and submit.

Reply:
Thanks, will do!